src/Adapter/Import/Handler/ImportHandlerFinder.php

```diff
@@ -41,7 +41,7 @@ final class ImportHandlerFinder implements ImportHandlerFinderInterface
     private $importHandlers;
 
     /**
-     * @param ImportHandlerInterface[] $importHandlers
+     * @param ImportHandlerInterface $importHandlers
```

Comment:
But it's an array 😮  it's the splat operator https://lornajane.net/posts/2014/php-5-6-and-the-splat-operator

Maybe PHPstan does not support the understanding  of the splat operator 🤔  ?

Reply:
Fixed.